ModulemdTranslationHelpers/cli.py

```diff
@@ -71,26 +71,36 @@ def cli(ctx, debug, branch, koji_url):
 
 
 @cli.command()
+
 @click.option('-p', '--pot-file',
               default='fedora-modularity-translations.pot',
               type=click.File(mode='wb', atomic=True, lazy=True),
               show_default=True,
               metavar="<PATH>",
               help="Path to the portable object template (POT) file to hold "
                    "the translatable strings.")
+
+@click.option('--project-name',
+              default='fedora-modularity-translations',
+              show_default=True,
+              help='Name of the project.')
+
 @click.pass_context
-def extract(ctx, pot_file):
+def extract(ctx, pot_file, project_name):
     """
     Extract translatable strings from modules.
-
     Extract translations from all modules included in a particular version of
     Fedora or EPEL.
     """
+    logger = logging.getLogger(__name__)
```

Comment:
See above: move this into `cli()` so it will be active for any and all operations, not just `extract()`.